From 47d338d3564b959ead91108f950eb410492ac28b Mon Sep 17 00:00:00 2001
From: crossbuild <crossbuild@localhost>
Date: Sun, 24 Nov 2024 20:52:50 +0000
Subject: [PATCH 01/10] arch/i386/kernel/dmi_scan.c tcc error:
 linux/include/asm/string.h:180: error: bad operand with opcode 'leal'

---
 linux-2.4.37/arch/i386/kernel/setup.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/linux-2.4.37/arch/i386/kernel/setup.c b/linux-2.4.37/arch/i386/kernel/setup.c
index 4f5f9dc..d37d972 100644
--- a/linux-2.4.37/arch/i386/kernel/setup.c
+++ b/linux-2.4.37/arch/i386/kernel/setup.c
@@ -168,7 +168,7 @@ struct e820map e820;
 unsigned char aux_device_present;
 
 extern void mcheck_init(struct cpuinfo_x86 *c);
-extern void dmi_scan_machine(void);
+//extern void dmi_scan_machine(void);
 extern int root_mountflags;
 extern char _text, _etext, _edata, _end;
 
@@ -1233,7 +1233,8 @@ void __init setup_arch(char **cmdline_p)
 #endif
 	paging_init();
 
-	dmi_scan_machine();
+	//tcc include/asm/string.h:180: error: bad operand with opcode 'leal'
+	//dmi_scan_machine();
 
 	/*
 	 * Parse the ACPI tables for possible boot-time SMP configuration.
-- 
2.35.1

